easybuild/easyconfigs/d/DIAMOND/DIAMOND-2.0.6-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'DIAMOND'
+version = '2.0.6'
+
+github_account = 'bbuchfink'
+homepage = 'https://github.com/%(github_account)s/%(namelower)s'
+description = """DIAMOND is a sequence aligner for protein and translated DNA searches, designed for high performance
+ analysis of big sequence data."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
```

Comment:
Please use the `GCC` toolchain instead of `GCCcore` + a dependency on `binutils`. It can be useful to have builds of `DIAMOND` with other compilers (_eg_ Intel) for performance reasons. Check for instance what we have for v0.9.30.

Reply:
@lexming If I change this `eb --check-contrib` says that: `Failed to determine toolchain hierarchy for GCC/7.3.0 when determining subtoolchain for dependency 'CMake': 'Could not find easyconfig for GCC toolchain version 7.3.0'` for `CMake` and `zlib`. I think the switch from `GCCcore` to `GCC` toolchain was done only for later `GCC` versions?

Reply:
@lexming any thoughts on this?

Reply:
@SethosII the version of GCC that corresponds to `GCCcore/7.3.0` is `GCC/7.3.0-2.30` (the binutils version was removed in later versions).

Reply:
I changed the toolchain.

Reply:
This has been done